readers-frontend/src/components/Muse.astro

```diff
@@ -0,0 +1,137 @@
+---
+import MuseComponent from "@solidComponents/MuseComponent";
+import { getMenuCategories } from "./utils/categories";
+import {
+  articlesWithArticleDrafts,
+  getArticleImage,
+  getArticles,
+} from "./utils/articles";
+import type { MuseNavbarCategory } from "./types";
+import Article from "./Article.astro";
+
+type articlesWithArticlesDraftAndLink = articlesWithArticleDrafts & {
+  imageUrl: string;
+};
+
+const museMenuCategories = (await getMenuCategories("muse")).filter(
+  (category) => category.categories_name !== "nouse"
+);
+
+museMenuCategories[0].categories_name = "muse";
+museMenuCategories[0].categories_displayName = "Features";
+
+const categories = museMenuCategories.reduce((accumulator, currentCategory) => {
+  accumulator.push({
+    name: currentCategory.categories_name,
+    displayName: currentCategory.categories_displayName
+      ? currentCategory.categories_displayName
+      : "",
+  });
+
+  return accumulator;
+}, [] as MuseNavbarCategory[]);
+
+const articles = new Map<string, articlesWithArticlesDraftAndLink[]>();
+
+for (let i = 0; i < museMenuCategories.length; i++) {
+  const featuredArticles = museMenuCategories[i]
+    .categories_featured!.split(",")
+    .map(Number);
+
+  const articlesList = (await getArticles(featuredArticles)).slice(0, 4);
+  const articlesArray = await Promise.all(
+    articlesList.map(async (article) => {
+      return {
+        ...article,
+        imageUrl: await getArticleImage(article),
+      };
+    })
+  );
+
+  articles.set(museMenuCategories[i].categories_name, articlesArray);
+}
+---
+
+<div class="relative mt-4 flex w-full flex-col bg-black" id="muse-banner">
+  <a href="/muse" class="flex w-full flex-row">
+    <img
+      class="mx-auto my-4 w-[27%] sm:w-[14%]"
+      src="https://bbcdn.nouse.co.uk/file/nouseSiteAssets/logo/MUSE%20Logo%20White%20small.png"
+    />
+  </a>
+
+  <MuseComponent client:only categories={categories}>
+    {
+      museMenuCategories.map((category, i) => (
+        <>
+          {/* Desktop */}
+          <div
+            class={`${
+              i === 0 ? "my-4 h-min opacity-100" : "h-0 opacity-0"
+            } hidden w-full flex-row px-[0.5%] transition-opacity delay-100 duration-700 sm:flex 2xl:px-[13%]`}
+            id={`muse_${category.categories_name}`}
+          >
+            {articles.get(category.categories_name)?.map((article, i) => (
+              <div class="flex w-1/4 flex-row">
+                {i !== 0 && (
+                  <span class="mx-3 h-full self-center border-r-[1px] border-white" />
```

Comment:
using spans inside the `w-1/4` div takes from the size of that article. Try using gaps and padding like in the featured articles.

<img width="1680" alt="Screenshot 2023-03-07 at 10 27 17 am" src="https://user-images.githubusercontent.com/58376513/223395610-ec14f182-a744-4d9d-ab9f-8d74062bbd41.png">

Reply:
Also the Muse black background is not the same as the design. It should take up all the width of the page